packages/flutter_test/lib/src/binding.dart

```diff
@@ -579,6 +579,10 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
       // our main future completing.
       assert(Zone.current == _parentZone);
       if (_pendingExceptionDetails != null) {
+        assert(
+          _unmangle(_pendingExceptionDetails.stack) == _pendingExceptionDetails.stack,
+          'The test binding presented an unmangled a test stack trace to the framework.',
```

Comment:
Change to `The framework was provided a mangled stack trace by a test binding`, the convention for assertion messages is to describe a true but unexpected state, afaik.

Reply:
This part isn't the framework though - it's in the test binding code.

Reply:
I was just confused by the grammar then, specifically the `a` after `unmangled`

Reply:
Ahh I see I'll clean that up